### DISCUSSION TITLE
Error: Couldn't find env file

### DISCUSSION THREAD

**Discussion by akozyritskiy** [RobertDWhite/WhiteMatterTech#8] [Q&A]

**Comment by RobertDWhite** [comment]
Hi @akozyritskiy 
Sorry for the delay. You probably have moved on by this point, but if you're still interested, please respond with which container you're chatting about and which .env file seems to be missing. Hopefully I can get you pointed in the right direction.

**Comment by RobertDWhite** [comment]
I am closing the discussion as stale. Please reopen if you come back and would like assistance troubleshooting. 

All the best!
